### PR TITLE
SED-1568 redesign step tree view

### PR DESCRIPTION
**Pull request by bojidaryovchev**
image

### CODE REVIEW

File: projects/step-frontend/src/lib/modules/execution/components/execution-tree/execution-tree.component.scss [hunk 1]
```diff
@@ -9,4 +9,8 @@ step-execution-tree {
   step-icon {
     background: transparent !important;
   }
+
+  step-tree {
+    --active-node-color: #5cb85c;
```
Comment (dvladir) [inline projects/step-frontend/src/lib/modules/execution/components/execution-tree/execution-tree.component.scss:14]: Should be set by scss variables, not css

File: projects/step-frontend/src/lib/modules/plan/components/plan-tree/plan-tree.component.scss [hunk 2]
```diff
@@ -3,3 +3,7 @@
 :host {
   @include core.full-height-block;
 }
+
+step-tree {
+  --active-node-color: #0082cb;
```
Comment (dvladir) [inline projects/step-frontend/src/lib/modules/plan/components/plan-tree/plan-tree.component.scss:8]: Should be set by scss variables, not css

File: projects/step-core/src/lib/modules/tree/components/tree/tree.component.scss [hunk 3]
```diff
@@ -1,5 +1,20 @@
 cdk-tree-node {
   display: flex;
+  position: relative;
+
+  .node-line {
+    position: absolute;
+    top: 0;
+    // level * paddingIdent + level * paddingMultiplier + (24px - 2px) / 2
+    left: calc(var(--level) * 24px + var(--level) * 8px + 11px);
```
Comment (dvladir) [inline projects/step-core/src/lib/modules/tree/components/tree/tree.component.scss:9]: Here css variable is suitable, as it take value from angular's binding. Looks fine.

**Review by dvladir — CHANGES_REQUESTED** [review]

**Review by dvladir — DISMISSED** [review]
@neogucky We need to deploy this branch, and check it visually

**Comment by neogucky** [comment]
@dvladir https://stepos-sed-1568-redesign.exense.ch/#/root/plans/editor/63aacdcd6388605bb6d3beff

**Review by neogucky — APPROVED** [review]
I like the new look!

**Review by dvladir — APPROVED** [review]

**Pull request merged**